File: VB/E3252_VB/Controllers/ServerModeLinqObsoleteController.vb
Imports System
Imports System.Web.Mvc

Public Class ServerModeLinqObsoleteController
    Inherits Controller
    Public Function IndexLinqObsolete() As ActionResult
        Return View()
    End Function
    Public Function GridViewPartialLinqObsolete() As ActionResult
        Return PartialView()
    End Function
End Class

File: CS/E3252_CS/Controllers/ServerModeLinqObsoleteController.cs
using System;
using System.Web.Mvc;

namespace E3252_CS.Controllers {
    public class ServerModeLinqObsoleteController : Controller {
        public ActionResult IndexLinqObsolete() {
            return View();
        }
        public ActionResult GridViewPartialLinqObsolete() {
            return PartialView();
        }
    }
}